App/AppInfo/appinfo.ini

```diff
@@ -19,8 +19,8 @@ Freeware=true
 CommercialUse=true
 
 [Version]
-PackageVersion=17.0.0.1
-DisplayVersion=18.0.1 Dev Test 1
+PackageVersion=18.0.2.1
```

Comment:
PackageVersion=18.0.1.1

Launcher doesn't need to be updated if you don't edit the OBSPortable.ini.